action.yml

```diff
@@ -94,9 +104,19 @@ runs:
         echo "Either the 'appSourcePath' or 'imageToDeploy' argument must be provided when providing the 'acrName' argument."
         exit 1
 
-    - name: Install pack CLI
+    - name: Install pack CLI on non-Windows runner
+      if: ${{ runner.os != 'Windows' }}
       shell: bash
-      run: (curl -sSL "https://github.com/buildpacks/pack/releases/download/v0.27.0/pack-v0.27.0-linux.tgz" | sudo tar -C /usr/local/bin/ --no-same-owner -xzv pack)
+      run: (curl -sSL "https://github.com/buildpacks/pack/releases/download/v0.27.0/pack-v0.27.0-${{ env.RUNNER_OS }}.tgz" | sudo tar -C /usr/local/bin/ --no-same-owner -xzv pack)
```

Comment:
Nice, looks like the [only possible values](https://docs.github.com/en/actions/learn-github-actions/contexts#runner-context) for this follow along with the [releases](https://github.com/buildpacks/pack/releases/tag/v0.27.0) naming conventions (besides casing, and casing doesn't matter). I assume that MacOS has curl and tar and uses `/user/local/bin` for path as well? I'm pretty unfamiliar with it

Reply:
If the runner os as macos is not supported / can't run these commands, then maybe we want to have a check in place that ensure that the runner OS is supported?

Reply:
but really this is just a nit

Reply:
>  I assume that MacOS has curl and tar and uses /usr/local/bin for path as well?

Yep that's correct -- the sample workflow I was testing in used a matrix of latest runners for each OS to make sure that the above commands worked where expected, so the `curl` and `tar` commands here work for macOS as well 👍